api/services/events/v1/events.proto

```diff
@@ -55,17 +54,9 @@ message PublishRequest {
 }
 
 message ForwardRequest {
-	Envelope envelope = 1;
+	containerd.types.Envelope envelope = 1;
```

Comment:
Just to confirm, since the structure is the same, the bytes on the wire should be the same. The protobuf type name changing won't have any impact on deserialization between v1 and v2?

Reply:
Message names aren't typically sent on the network (`Any` is an exception, but we don't use it here), protobuf cares about are message fields types/numbers to be compatible.

I googled a few confirmations just be sure:
- https://stackoverflow.com/questions/70975168/is-it-backwards-compatible-to-move-protos-to-another-file
- https://stackoverflow.com/questions/68239684/protobuf-is-it-backward-compatible-to-move-message-to-imported-file